subprojects/core/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -204,4 +205,19 @@
      */
     @Incubating
     void buildCache(Action<? super BuildCacheConfiguration> action);
+
+    /**
+     * Adds a action to be called to configure {@code PluginManagementSpec}
+     *
+     * @param pluginManagementSpec The action to execute
+     * @since 3.5
+     */
+    void pluginManagement(Action<? super PluginManagementSpec> pluginManagementSpec);
+
+    /**
+     * Get a {@link PluginManagementSpec} to configure how plugins will be applied.
+     *
+     * @return a {@link PluginManagementSpec}
+     */
+    PluginManagementSpec getPluginManagementSpec();
```

Comment:
so no get? I thought groovy let you access methods as fields when you did something like this.